src/Offer/ReadModel/JSONLD/PropertyPolyfillOfferRepository.php

```diff
@@ -241,6 +241,29 @@ function (array $json) {
         );
     }
 
+    private function removeNewLineLabels(JsonDocument $jsonDocument): JsonDocument
+    {
+        return $jsonDocument->applyAssoc(
+            function (array $json) {
+                $removeNewLineLabels = static function (array $json, string $propertyName): array {
+                    if (!isset($json[$propertyName]) || !is_array($json[$propertyName])) {
+                        return $json;
+                    }
+                    $json[$propertyName] = array_values(
+                        array_filter($json[$propertyName], fn ($label) => !str_contains($label, '\\n'))
```

Comment:
Maybe it is also possible that the string contains an Apple Style newline `\r`?

Reply:
`\r` was only used before Mac OS X though. `\r\n` is possible of course since that is still used on Windows, but that also contains `\n`.